lib/open_api_typesense/connection.ex

```diff
@@ -20,33 +20,52 @@ defmodule OpenApiTypesense.Connection do
   Setting new connection or using the default config.
 
   > #### On using this function {: .info}
-  > Functions e.g. `OpenApiTypesense.Health.health` don't need to explicitly pass this
+  > Functions e.g. `OpenApiTypesense.Health.health/0` don't need to explicitly pass this
   > unless you want to use another connection. Also, `api_key` is hidden when invoking
   > this function.
 
   ## Examples
+
       iex> alias OpenApiTypesense.Connection
-      
+
       iex> conn = Connection.new()
       %OpenApiTypesense.Connection{
         host: "localhost",
         port: 8108,
         scheme: "http",
         ...
       }
+
+      iex> Connection.new(%{})
+      ** (ArgumentError) Missing required fields: [:port, :scheme, :host, :api_key]
+          (open_api_typesense 0.2.0) lib/open_api_typesense/connection.ex:56: OpenApiTypesense.Connection.new/1
+      iex:2: (file)
+
   """
```

Comment:
For readability, it's best to add empty line after the `## Examples` and before the closing `"""` heredocs. Not sure if you want to adopt this to all comments in the project.